basiclti/tsugi-util/src/java/org/tsugi/http/HttpUtil.java

```diff
@@ -47,4 +49,21 @@ public static void printParameters(HttpServletRequest request) {
 		}
 	}
 
+	public static String getCookie(HttpServletRequest request, String lookup) {
+		if ( request == null || lookup == null ) return null;
+
+		// https://stackoverflow.com/questions/11047548/getting-cookie-in-servlet
+		Cookie[] cookies = request.getCookies();
+		if ( cookies == null ) return null;
+		for (int i = 0; i < cookies.length; i++) {
```

Comment:
For-each loop would save a line of code and cleaner here since you don't need the i variable

for (Cookie cookie: cookies) {